rails_event_store/spec/active_job_dispatcher_spec.rb

```diff
@@ -41,42 +41,60 @@ module RailsEventStore
 
     it "builds async proxy for ActiveJob::Base ancestors" do
       expect_to_have_enqueued_job(AsyncHandler) do
-        ActiveJobDispatcher.new.call(AsyncHandler, event)
+        ActiveJobDispatcher.new.call(AsyncHandler, [event, serialized_event])
       end
       expect(AsyncHandler.received).to be_nil
       perform_enqueued_jobs(AsyncHandler.queue_adapter)
-      expect(AsyncHandler.received).to eq(yaml)
+      expect(AsyncHandler.received).to eq({
+        "event_id"        => "83c3187f-84f6-4da7-8206-73af5aca7cc8",
+        "data"            => "--- {}\n",
+        "metadata"        => "--- {}\n",
+        "event_type"      => "RubyEventStore::Event",
+        "_aj_symbol_keys" => ["event_id", "data", "metadata", "event_type"]
+      })
```

Comment:
This assertion does not seem valuable to me. I would prefer the handler to deserialize the event and we should compare that what we received is equal to `event` and has the same `metadata` (checking metadata is not part of == definition for events)

Reply:
Do we really care about event deserialization at this point ? Isn't this more about right stuff being received through ActiveJob layer ?